packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -792,7 +793,7 @@ class Auth {
      * A wrapper method for commerce-sdk-isomorphic helper: loginGuestUser.
      *
      */
-    async loginGuestUser() {
+    async loginGuestUser(parameters: helpers.CustomQueryParameters = {}) {
```

Comment:
This should be optional
```suggestion
    async loginGuestUser(parameters?: helpers.CustomQueryParameters = {}) {
```